fit.py

```diff
@@ -0,0 +1,296 @@
+import os
+import re
+
+import pymongo
+from bson.objectid import ObjectId
+from dotenv import load_dotenv
+from flask import (
+    Flask,
+    flash,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
+from flask_login import LoginManager
+from utilities import paginate
+from validators import validate_recipe
+
+load_dotenv()
+
+app = Flask(__name__)
+# Get secret key
+app.config["SECRET_KEY"] = "qwert"
+login = LoginManager(app)
+
+# Mongo Database for FITNESS POT
+myclient = pymongo.MongoClient(os.environ.get("MONGO_CLUSTER"))
```

Comment:
What you have in your .env is `MONGO_CLUSTER`, why are asking user to add `MONGO_URI` to Heroku?